src/test/java/de/tum/in/www1/artemis/exam/ExamIntegrationTest.java

```diff
@@ -2107,6 +2104,7 @@ private void configureCourseAsBonusWithIndividualAndTeamResults(Course course, G
     @CsvSource({ "false, false", "true, false", "false, true", "true, true" })
     @WithMockUser(username = TEST_PREFIX + "instructor1", roles = "INSTRUCTOR")
     void testGetExamScore(boolean withCourseBonus, boolean withSecondCorrectionAndStarted) throws Exception {
+        gradingScaleRepository.deleteAll();
```

Comment:
this line seems to cause H2 Tests to fail (https://bamboo.ase.in.tum.de/browse/ARTEMIS-TESTS5075-JAVATEST-1) with `EntityNotFoundException`

Reply:
@laadvo I removed it for now, although I couldn't reproduce it locally.